datafusion/sqllogictest/test_files/regexp/init_data.slt.part

```diff
@@ -0,0 +1,30 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+statement ok
+create table regexp_test_data (str varchar, pattern varchar, start int, flags varchar) as values
+    ('abc',       '^(a)', 1, 'i'),
```

Comment:
```suggestion
    (NULL,       '^(a)', 1, 'i'),
    ('abc',       '^(a)', 1, 'i'),
```
I found that we have the null test for all the regex functions. We could add the null data to the dataset. It can ensure that any case using this table needs to care about the null problem.